applications/openshift/api-server/audit_log_forwarding_enabled/rule.yml

```diff
@@ -7,7 +7,7 @@ description: |-
   The cluster-logging-operator is able to do this with the <pre>ClusterLogForwarders</pre> resource.
   The forementioned resource can be configured to logs to different third party systems.
   For more information on this, please reference the official documentation:
-  {{{ weblink(link="https://docs.openshift.com/container-platform/4.6/logging/cluster-logging-external.html") }}}
+  {{{ weblink(link="https://docs.openshift.com/container-platform/4.16/observability/logging/log_collection_forwarding/configuring-log-forwarding.html") }}}
```

Comment:
nit: we can future proof this by using `latest/observability` instead of `4.16`.